windows/src/engine/kmcomapi/processes/keyboard/kpinstallkeyboard.pas

```diff
@@ -64,11 +64,12 @@ interface
     ikLegacyRegisterAndInstallProfiles);
 
   TKPInstallKeyboard = class(TKPBase)
-    procedure Execute(const FileName, PackageName: string; FInstallOptions: TKPInstallKeyboardOptions; Languages: TPackageKeyboardLanguageList; Force: Boolean);
+    procedure Execute(const FileName, PackageID: string; FInstallOptions: TKPInstallKeyboardOptions; Languages: TPackageKeyboardLanguageList; Force: Boolean);
+    procedure RegisterProfiles(const FileName, PackageID: string; FInstallOptions: TKPInstallKeyboardOptions; PackageLanguageMetadata: TPackageKeyboardLanguageList);
```

Comment:
This function was refactored out of the `Execute` function and changed just slightly.